### ISSUE TITLE
view privileges might not be correctly generated

### ISSUE THREAD

**Issue by ruslantalpa** [subzerocloud/apgdiff#8]

**Comment by steve-chavez** [comment]
Needs clarification, on the latest version:
```sql
-- v1.sql

create table items(id integer, name text);
create view items_view as select id from items;

-- v2.sql

create table items(id integer, name text);
create view items_view as select id from items;
grant select, insert, update on items_view to admin;

-- apgdiff v1.sql v2.sql

REVOKE ALL ON TABLE items_view FROM admin;
GRANT SELECT, INSERT, UPDATE ON TABLE items_view TO admin;
```

The diff is valid and the view's privileges are assigned correctly despite having the `TABLE` keyword because that's optional.

**Comment by ruslantalpa** [comment]
i showed an example in slack

**Comment by steve-chavez** [comment]
A couple of errors I found on view privileges on specific columns:
```sql
-- v1.sql

create table items(id integer, name text);
create view items_view as select id from items;

-- v2.sql

create table items(id integer, name text);
create view items_view as select id from items;
grant select(id, name), insert(name), update on items_view to admin;

-- diff.sql

GRANT SELECT, INSERT, UPDATE ON TABLE items_view TO admin;
```
i.e. When having a last privilege without cols the previous col privileges get ignored.
```sql
-- v1.sql

create table items(id integer, name text);
create view items_view as select id from items;

-- v2.sql

create table items(id integer, name text);
create view items_view as select id from items;
grant select(id, name), insert, update(name) on items_view to admin;

-- diff.sql
-- gives java.lang.NullPointerException
```
When having a privilege without cols followed by a privilege with cols the parsing fails.

**Comment by steve-chavez** [comment]
This is fixed on https://github.com/steve-chavez/apgdiff/commit/dab3bcd9c32a01cc9555c71e3b06a5629cdc83cb.

I'll do a PR when we are in sync with upstream.

**Comment by steve-chavez** [comment]
This is already merged in the `subzero` branch and released in `2.5.3-subzero` so I'll close the issue.
Is not merged in upstream though, PR is open https://github.com/fordfrog/apgdiff/pull/227.
